response/response.go

```diff
@@ -65,44 +61,32 @@ func New(code int, message string, data *Data) *Response {
 	}
 }
 
-// mySQLError is an almost identical copy from the mysql errors coming from our driver
-type mySQLError struct {
-	Number  uint16
-	Message string
+// DBError returns a prepared 503 Service Unavailable response.
+func DBError(err error) *Response {
+	return DBErrorf("", err)
 }
 
-func (m *mySQLError) Error() string {
-	return fmt.Sprintf("Error %d: %s", m.Number, m.Message)
+// DBErrorf returns a prepared 503 Service Unavailable response,
+// using the user provided formatted message.
+func DBErrorf(format string, err error) *Response {
+	var msg string
+	switch format {
+	case "":
+		msg = fmt.Sprintf("db error: %v", err)
+	default:
+		msg = fmt.Sprintf(format, err)
+	}
+	return New(http.StatusServiceUnavailable, msg, nil)
 }
 
-// SQLError returns a prepared 204 No Content response if the error passed is of type sql.ErrNoRows,
-// otherwise, returns a 500 Internal Server Error prepared response.
+// SQLError - DEPRECATED currently only wraps DBError
```

Comment:
Godoc convention for deprecation is as follows:

```golang
// First comment line here.
//
// Deprecated: Short description of why this is deprecated
func SQLError(err error) *Response {}
```

https://blog.golang.org/godoc-documenting-go-code

Reply:
I was not aware of this. Good catch! Fixed it.